adapters/appnexus/appnexus.go

```diff
@@ -223,7 +215,11 @@ func splitRequests(imps []openrtb2.Imp, request *openrtb2.BidRequest, requestExt
 	headers.Add("Content-Type", "application/json;charset=utf-8")
 	headers.Add("Accept", "application/json")
 
-	marshalAndSetRequestExt(request, requestExtension, errs)
+	var err error
+	request.Ext, err = json.Marshal(requestExtension)
+	if err != nil {
+		errs = append(errs, err)
+	}
```

Comment:
nitpick - I think this could also be written as - 
```
if request.Ext, err := json.Marshal(requestExtension); err != nil {
         errs = append(errs, err)
}
```